kubetest2-tf/Makefile

```diff
@@ -31,3 +45,7 @@ install-deployer-tf: install-prereq
 	$(INSTALL) -d $(INSTALL_DIR)
 	$(INSTALL) $(OUT_DIR)/$(BINARY_NAME) $(INSTALL_DIR)/$(BINARY_NAME)
 .PHONY: install-deployer-tf
+
+build-deployer-tf:
```

Comment:
can we keep it simple as `build` because it is already inside the directory.

Reply:
addressed the suggestion! thanks!